cpg_infra/config.py

```diff
@@ -166,6 +166,7 @@ class GCP(DeserializableDataclass):
     # delete, which can end up with missing permissions. To implement the first phase
     # (delete), simply change this to 'True', then revert to reapply group memberships
     disable_group_memberships: bool = False
+    group_prefix: str = ''
```

Comment:
Could you add a comment here describing when it's useful to set this?
(Could maybe default this to `None` too?)

Reply:
Resolved :)